src/core/src/sim/mc/monte_carlo.rs

```diff
@@ -64,10 +67,21 @@ impl MonteCarlo {
                 we can not add new moves."
             );
         }
-        self.moves.push(mcmove);
+        self.moves.push((mcmove, MoveCounter::new(None)));
         self.frequencies.push(frequency);
     }
 
+    pub fn add_with_acceptance(&mut self, mcmove: Box<MCMove>, frequency: f64, target: Option<f64>) {
```

Comment:
I feel like the name of this function should start with `add_move`, but I can not find a nice one. I though about `add_move_with_acceptance`, `add_move_auto_update`, ... 

What do you think?